packages/big-design/src/components/ToggleButtons/ToggleButton/spec.tsx

```diff
@@ -0,0 +1,48 @@
+import { fireEvent, render } from '@test/utils';
+import 'jest-styled-components';
+import React from 'react';
+
+import { ToggleButton } from './ToggleButton';
+
+test('render selected and enabled', () => {
+  const { container } = render(<ToggleButton label="This is a selected toggle button" selected={true}></ToggleButton>);
```

Comment:
When you get rid of label, you'll probably want to push it here:
```suggestion
  const { container } = render(<ToggleButton selected={true}>This is a selected toggle button</ToggleButton>);
```